src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -1064,6 +1064,7 @@
   "LastDuration": "Last Duration",
   "LastExecution": "Last Execution",
   "LastUsed": "Last Used",
+  "LastSearchTime": "Last Search",
```

Comment:
If this is for width reasons then the key should be renamed as well.

Reply:
The value is called LastSearchTime consistently on the backend, hence why I kept the key in this way.

On the frontend we show a time if the search is from today, or a date if it‘s before today. I felt that the column name „Last Search Time“ doesn‘t do it justice since when showing dates there is no time.

It‘s a detail and I can totally live with „Last Search Time“ as column header - what do you think?

Reply:
Yeah, but this is purely for the translation in the UI, makes sense to show `Last Search` in the UI, but the key for the translation should match in this case.

Reply:
Will amend the key 👍